Adding Klaytn, Dogechain, DFK, and Cronos

simonmahns opened the pull request:
## Description

Added Klaytn, Dogechain, DFK, and Cronos, ammended an error in the boba file.

tmm commented on packages/chains/src/cronos.ts:
```diff
@@ -0,0 +1,26 @@
+      public: { http: ['https://node.croswap.com/rpc'] },
+    },
+    blockExplorers: {
+      etherscan: { name: 'CronosScan', url: 'https://cronoscan.com/' },
+      default: { name: 'CronosScan', url: 'https://cronoscan.com/' },
```
```suggestion
      etherscan: { name: 'CronosScan', url: 'https://cronoscan.com' },
      default: { name: 'CronosScan', url: 'https://cronoscan.com' },
```

tmm commented on packages/chains/src/dfk.ts:
```diff
@@ -0,0 +1,20 @@
+import { Chain } from './types'
+
+export const dfk = {
+    id: 53935,
```
```suggestion
    id: 53_935,
```

tmm commented on packages/chains/src/dfk.ts:
```diff
@@ -0,0 +1,20 @@
+      symbol: 'JEWEL',
+    },
+    rpcUrls: {
+      default: { http: ['https://subnets.avax.network/defi-kingdoms/dfk-chain/rpc	'] },
+      public: { http: ['https://subnets.avax.network/defi-kingdoms/dfk-chain/rpc	'] },
```
```suggestion
      default: { http: ['https://subnets.avax.network/defi-kingdoms/dfk-chain/rpc'] },
      public: { http: ['https://subnets.avax.network/defi-kingdoms/dfk-chain/rpc'] },
```

tmm commented on packages/chains/src/dogechain.ts:
```diff
@@ -0,0 +1,22 @@
+import { Chain } from './types'
+
+export const dogechain = {
+    id: 2000,
```
```suggestion
    id: 2_000,
```

tmm commented on packages/chains/src/dogechain.ts:
```diff
@@ -0,0 +1,22 @@
+      symbol: 'DC',
+    },
+    rpcUrls: {
+      default: { http: ['https://rpc.dogechain.dog	'] },
```
```suggestion
      default: { http: ['https://rpc.dogechain.dog'] },
```

tmm commented on packages/chains/src/klaytn.ts:
```diff
@@ -0,0 +1,21 @@
+      symbol: 'KLAY',
+    },
+    rpcUrls: {
+      default: { http: ['https://cypress.fautor.app/archive	'] },
+      public: { http: ['https://cypress.fautor.app/archive	'] },
```
```suggestion
      default: { http: ['https://cypress.fautor.app/archive'] },
      public: { http: ['https://cypress.fautor.app/archive'] },
```

tmm commented on packages/chains/src/klaytn.ts:
```diff
@@ -0,0 +1,21 @@
+import { Chain } from './types'
+
+export const klaytn = {
+    id: 8217,
```
```suggestion
    id: 8_217,
```

tmm commented on packages/chains/src/klaytn.ts:
```diff
@@ -0,0 +1,21 @@
+      public: { http: ['https://cypress.fautor.app/archive	'] },
+    },
+    blockExplorers: {
+      etherscan: { name: 'KlaytnScope', url: 'https://scope.klaytn.com/' },
+      default: { name: 'KlaytnScope', url: 'https://scope.klaytn.com/' },
```
```suggestion
      etherscan: { name: 'KlaytnScope', url: 'https://scope.klaytn.com' },
      default: { name: 'KlaytnScope', url: 'https://scope.klaytn.com' },
```

tmm reviewed (changes requested):
Please add a patch [changeset](https://github.com/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md#versioning)

simonmahns commented:
> Please add a patch [changeset](https://github.com/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md#versioning)

Hey, added change set (first time using, cool software!) and fixed changes. Thanks for the feedback.

tmm reviewed (changes requested):
Please make sure all these chains are exported from the `index.ts` file
